docs/secure_manager.md

```diff
@@ -273,3 +278,52 @@ users = append(users, User{Name: "Member", Role: "member"})
 ```
 
 ---
+
+## 5. TLS
+### 5.1 Description
+The **[Transport Layer Security (TLS)](https://en.wikipedia.org/wiki/Transport_Layer_Security)** is a cryptographic protocol designed to provide communications security over a computer network. The Edge-orchestration project uses a standard package [crypto/tls](https://pkg.go.dev/crypto/tls#pkg-overview). The product uses an authentication mechanism with a center authority, realizing a certificate chain of trust and data encryption based on a session key.
+
+---
+
+### 5.2 Workflow
+> To view the Workflow, you need to install a `plantuml` extension for your browser.
+ 
+```plantuml
+
+@startuml
+  autonumber
+  CA->>CA: Generate CA Root Certificate ca.crt ca.key
+  "Node (192.168.0.100)"->>"Node (192.168.0.100)": Generate private key hen.key, request hen.csr
+  "Node (192.168.0.100)"->>CA: Send hen.csr to CA
+  CA-->>"Node (192.168.0.100)": Send ca.crt and hen.crt to Node
+  Note right of CA: For each Node, must be executed items 2-4
+@enduml
+```
+ 
+---
+
+### 5.3 Generation key infrastructure
+
+Before starting work the _Home Edge Network_, need to create a key infrastructure (for a certification authority and for each node separately).
+
+There are two scripts: 
+ * [tools/gen_ca_cert.sh](../tools/gen_ca_cert.sh) for generation of CA Root Certificate, that consists of a private key `ca.key` and a self-signed root certificate `ca.crt`
+ * [tools/gen_hen_cert.sh](../tools/gen_hen_cert.sh) for generation of Home Edge Node (HEN) Certificate, that consists of a private key `hen.key` and a signature HEN certificate: `hen.crt`
+ 
+#### CA Root Certificate
+Generating a CA Root Certificate only needs to be done once for _Home Edge Network_ by running command:
+```shell
+$ gen_ca_cert.sh
```

Comment:
How about changing the base folder in accordance with the above example rather than tools? 

```suggestion
$ . tools/gen_ca_cert.sh
```